pkg/apis/stork/v1alpha1/applicationbackup.go

```diff
@@ -66,6 +67,7 @@ type ApplicationBackupStatus struct {
 	LastUpdateTimestamp metav1.Time                      `json:"lastUpdateTimestamp"`
 	FinishTimestamp     metav1.Time                      `json:"finishTimestamp"`
 	TotalSize           uint64                           `json:"totalSize"`
+	Namespaces          []string                         `json:"namespaces"`
```

Comment:
We already have the namespaces list in the applicationbackup spec definition itself. Cann't we us that list itself?

Reply:
During reconcile at px-backup we updated the datastore object with the CR status. So added the same in application backup status too.
Will verify the same once